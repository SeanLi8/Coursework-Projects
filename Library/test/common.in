add collection books.csv
add member Sean Li
add member Neas Il
common 999999 888888
common 100000 999999 
common 100000 100001
rent 100000 111122
rent 100000 111121
rent 100000 111125
rent 100000 111126
rent 100000 111127
relinquish all 100000
rent 100001 111125
rent 100001 111126
rent 100001 111127
relinquish all 100001
common 100000 100001
rent 100000 111111
rent 100001 111134
relinquish all 100000
relinquish all 100001
member history 100000
member history 100001
common 100000 100001
exit

