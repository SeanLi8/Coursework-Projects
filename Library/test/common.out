user: 26 books successfully added.

user: Success.

user: Success.

user: No such member in system.

user: No such member in system.

user: No common books.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Biggles in the Battle (W.E. Johns)
Aeneid (Virgil)
Halloween (Jamie Lee Curtis)

user: Success.

user: Success.

user: Success.

user: Success.

user: Harry Potter 5 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Biggles in the Battle (W.E. Johns)
Aeneid (Virgil)
Halloween (Jamie Lee Curtis)
To Kill a Mockingbird (Harper Lee)

user: Biggles in the Battle (W.E. Johns)
Aeneid (Virgil)
Halloween (Jamie Lee Curtis)
To Kill a Mockingbird (Harper Lee)

user: Biggles in the Battle (W.E. Johns)
Aeneid (Virgil)
Halloween (Jamie Lee Curtis)

user: Ending Library process.
